public/css/admwswp-public.css

```diff
@@ -2,10 +2,61 @@
  * All of the CSS for your public-facing functionality should be
  * included in this file.
  */
-.admwswp-addToCart {
+.admwswp-loader {
+  display: none;
+  visibility: hidden;
+}
+
+.admwswp-dialog .admwswp-eventRequestPopup .admwswp-loader {
+  display: block;
+  visibility: visible;
+}
+
+.admwswp-btn.admwswp-waitWeblink {
+  opacity: 0.8;
+  cursor: default !important;
+  background-color: #6c757d !important;
```

Comment:
are the `important` needed? cant we go down the tree to avoid it

I really prefer to avoid it unless somebody else is using it and we need to do it to overwrite it